modular_splurt/code/modules/clothing/clothing.dm

```diff
@@ -0,0 +1,53 @@
+/obj/item/clothing
+	var/last_bites = 3 //Once clothes is shredded,
+
+/obj/item/reagent_containers/food/snacks/clothing
+	name = "oops"
+	desc = "If you're reading this it means I messed up. This is related to moths eating clothes and I didn't know a better way to do it than making a new food object."
+	list_reagents = list(/datum/reagent/consumable/nutriment = 8) //Cloth eaters get some nutrients.
+	tastes = list("dust" = 1, "lint" = 1)
+
+//A call on attemp_forcefeed() without async to properly know if it worked or not. In theory this shouldn't cause any issues as only a small part of the population should ever run this.VS normal eating.
+/obj/item/reagent_containers/food/snacks/clothing/attack(mob/living/M, mob/living/user, attackchain_flags = NONE, damage_multiplier = 1)
+	if(user.a_intent == INTENT_HARM)
+		return ..()
+	return attempt_forcefeed(M, user)
+
+
+//As a bonus for having the Cloth Eater trait. You gain extra mood from eatin clothes, but damage them at the same time.
+/obj/item/clothing/attack(mob/M, mob/user, def_zone)
+	if(user.a_intent != INTENT_HARM)
+		if(HAS_TRAIT(M,TRAIT_CLOTH_EATER) || isinsect(M))
+			if(limb_integrity == 0)
+				to_chat(M, "<span class='warning'>This item is too tough to eat.")
+				return FALSE
```

Comment:
Maybe make the early returns in this proc return ..()? Or is there any reason why it doesn't?

Reply:
Mistake on my end. Original intent was to most likely prevent the player from smacking themselves. But ..() already does that.

Reply:
The main reason for doing this instead of using a ..() was to make sure a player didn't hit themselves with anything while trying to eat different outfits. 

I'm not sure any can do damage, but just incase one somehow slips in and does. This should hopefully prevent that.
I can use  ..() but each failed attempt due to an outfit not being edible will result in them bapping themselves with it.

My original message was wrong, and I edited it to clarify that. But wasn't sure the edit was seen. So I made this message and deleted the old one. Sorry for the spam.